.github/workflows/draft-new-release.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
           git add CHANGELOG.md gradle.properties $(find charts -name Chart.yaml) $(find charts -name README.md)
           git commit --message "Prepare release ${{ github.event.inputs.version }}"
 
-          echo "::set-output name=commit::$(git rev-parse HEAD)"
+          echo "{commit}={$(git rev-parse HEAD}" >> $GITHUB_OUTPUT
```

Comment:
forgot the closing brace

Reply:
Ah I see, I'll update those changes. Thank you.